rotkehlchen/tests/unit/decoders/test_optimism_bridge.py

```diff
@@ -100,7 +100,7 @@ def test_deposit_eth(database, ethereum_inquirer, ethereum_accounts):
     ]
 
 
-@pytest.mark.vcr()
+# @pytest.mark.vcr()
```

Comment:
Why all these removing of VCR here? Changes order of queries eh?

Reply:
hmm, after the review changes there is no need for new cassetes.

Reply:
I think there is